### ISSUE TITLE
websocket does not establish under 'server' mode.

### ISSUE THREAD

**Issue by gaokun** [webpack-contrib/webpack-bundle-analyzer#312]
#### Issue description
websocket **does not** establish under 'server' mode.

```js
// yes, 'server' is default value of 'analyzerMode'
new BundleAnalyzerPlugin({analyzerMode: 'server'})
```
> I am using `webpack-bundle-analyzer` as  webpack plugin.

I have researched. This issue is related PR #215.

#### Testing
In below code, you can find that `reporter.html` uses `enableWebSocket` before setting.

### viewer.jsx
```js
console.log('use enableWebSocket', window.enableWebSocket);
// Initializing WebSocket for live treemap updates
let ws;
try {
  if (window.enableWebSocket) {
    ws = new WebSocket(`ws://${location.host}`);
  }
} catch (err) {
  console.warn(
    "Couldn't connect to analyzer websocket server so you'll have to reload page manually to see updates in the treemap"
  );
}
```

### viewer.ejs
```html
  <%- include('script', { filename: 'viewer.js' }) %>
  </head>
  <body>
    <div id="app"></div>
    <script>
      window.chartData = <%- escapeJson(chartData) %>;
      window.defaultSizes = <%- escapeJson(defaultSizes) %>;
      window.enableWebSocket = <%- escapeJson(enableWebSocket) %>;
      console.log('set Websocket', window.enableWebSocket);
    </script>
```

Test result in browser.

![image](https://user-images.githubusercontent.com/7044628/65318808-e8631980-dbd0-11e9-9842-93c90e3d7f6a.png)

**I am making a PR to fix it.**

#### Technical info
* Webpack Bundle Analyzer version: 3.5.0
* Webpack version: 4.39.3
* Node.js version: 10.15.3
* npm/yarn version: 6.11.3
* OS: MacOS

**Comment by valscion** [comment]
Thanks for the detailed issue report. Left a review to the attached PR :relaxed:

**Comment by valscion** [comment]
Thanks for the contributions :relaxed: The fix has been released in `v3.5.1`
